scripts/sideload.sh

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env bash
+
+ #
+ # Copyright (c) 2024 Project CHIP Authors
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+#!/bin/bash
+ROOT_DIR=$(realpath $(dirname "$0")/..)
+SCRIPT_DIR="$ROOT_DIR/scripts"
+
+
+DATE_STR=$(date +"%F-%H-%M-%S")
+
+SIDELOAD_LOGFILE_PATH="logs/sideload_$DATE_STR.log"
+
+# Redirect all output (stdout and stderr) to both the terminal and the log file
+exec > >(tee -a "$SIDELOAD_LOGFILE_PATH") 2>&1
```

Comment:
I'm not familiar with this redirect "`> >`" with space.

Did you mean:
```suggestion
exec >>(tee -a "$SIDELOAD_LOGFILE_PATH") 2>&1
```

Let me know

Reply:
This space is mandatory when you want to display the log both on the terminal and in a file. https://stackoverflow.com/questions/49509264/explain-the-bash-command-exec-tee-log-file-21.

**In summary:**
`exec >> file | Redirects all output to the file. Nothing is displayed on the terminal.`
`exec > >(tee file) | Sends output both to the terminal and to the file at the same time.`